web/static_files.go

```diff
@@ -0,0 +1,99 @@
+// +build bindata
```

Comment:
I'd put the handlers inside `handler` package as in our other current web apps.

Reply:
`handler` is a quite common name and in other apps that definitely makes sense because all the app is http server. In lookout http is just a very small part.

Let's start with only 1 package for now and split it later if we need. WDYT?

Reply:
I always prefer standard ways to face similar problems and to lower the differences between similar apps as much as possible; Anyway, it is not a strong opinion, just a personal preference that can be ignored if it does not suit you :wink:

Reply:
Okay. Let's keep it for now like this if you don't feel too strongly about it. And we can change it later if needed.

Just to explain my concern better:

The problem with "the standard" way is the context.
For example in gitbase-web top level `server` package means http server. But in lookout it doesn't. Same here, when I read lookout source code `web.Static` is obvious but `handler.Static` not really. Is it http handler? Or is it github event handler?